src/main/java/TaskList.java

```diff
@@ -0,0 +1,127 @@
+import java.time.LocalDate;
+import java.time.LocalTime;
+import java.time.format.DateTimeFormatter;
+import java.util.ArrayList;
+
+/**
+ * A List that stores all Task created.
+ */
+public class TaskList {
+
+    ArrayList<Task> tasks;
+    Storage taskStorage;
+
+    /**
+     * To instantiate the task list from storage, loading all tasks that are currently in the data.txt.
+     * @param taskStorage storage where the copy of the task file is stored.
+     */
+    public TaskList(Storage taskStorage) {
+
+        this.tasks = new ArrayList<Task>();
+        this.taskStorage = taskStorage;
+        ArrayList<String> tasklist = taskStorage.startupStorage();
+        createStartUpList(tasklist);
+
+    }
+
+
+    /**
+     * Method invoked to create the initial start-up list.
+     * @param tasklist list that is loaded with the tasks from data.txt from the file.
+     */
+    private void createStartUpList(ArrayList<String> tasklist) {
+
+        for (String task : tasklist) {
+
+            String[] tasksArr = task.split("\\|");
+            String taskType = tasksArr[0].trim();
+            String taskCompleted = tasksArr[1].trim();
+            String taskAction = tasksArr[2].trim() + " ";
+
+            if (taskType.equals("T")) {
```

Comment:
You may consider making the task type more explicit and clear to read, perhaps by writing out `if (taskType.equals("todo"))`, and also if `if (taskCompleted)` would have a better flow :)